src/diffusers/pipelines/controlnet/pipeline_controlnet.py

```diff
@@ -659,7 +656,9 @@ def check_inputs(
         ):
             if isinstance(controlnet_conditioning_scale, list):
                 if any(isinstance(i, list) for i in controlnet_conditioning_scale):
-                    raise ValueError("A single batch of multiple conditionings is not supported at the moment.")
+                    raise ValueError(
+                        "A single batch of varied conditioning scale settings is not supported at the moment."
+                    )
```

Comment:
While we're at it, could we provide a little example in the error message? Something like:

```
"A single batch of varied conditioning scale settings (ex: ...) is not supported at the moment."
```